Code/Framework/AzFramework/AzFramework/Windowing/NativeWindow.cpp

```diff
@@ -10,6 +10,17 @@
 
 #include <AzCore/Console/IConsole.h>
 
+void OnVsyncIntervalChanged(uint32_t const& interval)
+{
+    AzFramework::WindowNotificationBus::Broadcast(
+        &AzFramework::WindowNotificationBus::Events::OnVsyncIntervalChanged, AZ::GetClamp(interval, 0u, 4u));
+}
+
+// NOTE: On change, broadcasts the new requested vsync interval to all windows.
+// The value of the vsync interval is constrained between 0 and 4
+// Vsync intervals greater than 1 are not currently supported on the Vulkan RHI (see #2061 for discussion)
+AZ_CVAR(uint32_t, rpi_vsync_interval, 1, OnVsyncIntervalChanged, AZ::ConsoleFunctorFlags::Null, "Set swapchain vsync interval");
```

Comment:
Should we change the prefix of the cvar here since this isn't in Atom anymore?

Reply:
I will update.